pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -610,10 +621,27 @@ func (c *clusterAutoscaler) computeShootResourcesData(serviceAccountName string)
 		}
 	)
 
-	return registry.AddAllAndSerialize(
-		clusterRole,
-		clusterRoleBinding,
-		role,
-		rolebinding,
-	)
+	objects := []client.Object{clusterRole, clusterRoleBinding, role, rolebinding}
+
+	if c.hasExpanderConfigMapConfigured() {
+		annotations := map[string]string{}
+		if c.expanderConfigMap.Annotations != nil {
+			annotations = c.expanderConfigMap.Annotations
+		}
+		annotations[resourcesv1alpha1.Ignore] = "true"
+
+		expanderConfigMap := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        clusterAutoScalerPriorityConfigMapName,
+				Namespace:   metav1.NamespaceSystem,
+				Labels:      c.expanderConfigMap.Labels,
+				Annotations: annotations,
+			},
+			Data: c.expanderConfigMap.Data,
```

Comment:
I don't think we should copy all data - just the one necessary for CA. 
```suggestion
			Data: map[string]string{"priorities": *c.expanderConfiguration},
```